backend/src/services/calendar/api/methods.py

```
@@ -0,0 +1,2907 @@
+                        "errors": [
+                            {
+                                "domain": "global",
+                                "reason": "parseError",
+                                "message": "Invalid JSON",
+                            }
+                        ],
+                    }
+                },
+                status_code=status.HTTP_400_BAD_REQUEST,
+            )
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)